Dockerfile/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.8-slim-buster
+
+RUN pip install --upgrade pip
```

Comment:
can removed you should use a latest image where the latest python and pip version is included.

Reply:
Thank you!
I have deleted this line and it's working perfectly!